README.md

```diff
@@ -28,7 +28,7 @@ Go to the official [Code For Life website][c4l].
     * `sudo apt-get install ruby2.0` - still Ruby 1.9 hiding under `ruby` command.
     * `sudo gem install sass -v 3.3.4` - later versions incompatible with Ruby 1.9 (see above).
 
-* Clone the repo: `https://github.com/ocadotechnology/rapid-router.git`
+* Clone the repo: `https://github.com/ocadotechnology/rapid-router.git`. Fork it first if you want to contribute, or work on a separate branch.
```

Comment:
You wouldn't if you're not part of / contributor to the organisation, so that' s a valid point thanks